flash/image/detection/model.py

```diff
@@ -49,7 +49,7 @@ def __init__(
         pretrained: bool = True,
         optimizer: OPTIMIZER_TYPE = "Adam",
         lr_scheduler: LR_SCHEDULER_TYPE = None,
-        learning_rate: float = 5e-3,
+        learning_rate: float = 1e-2,
```

Comment:
@ethanwharris Can you elaborate why the default learning rate has been changed here? Thanks.

Reply:
Hey, @ligaz sorry just an accidental change 😃 Woud you prefer the old default? Or something else? Please feel free to open a PR to revert / update it

Reply:
It was a general question what's the reason behind this change.

We are defaulting the optimizer to `Adam` so my suggestion is to use the default learning rate for it. From the PyTorch docs [here](https://pytorch.org/docs/stable/generated/torch.optim.Adam.html):
```
learning rate (default: 1e-3)
```

Reply:
I think that makes sense! I wonder if there's a way we could just not provide the LR to the optimizer constructor if it is `None`? That way if you don't override the LR you would always just get the default for your chosen optimizer. What do you think?

Reply:
I was thinking about the same. It turns out that it defaults to the code in the core Task class which uses `5e-5` as a default value. This is the relevant [code](https://github.com/PyTorchLightning/lightning-flash/blob/master/flash/core/model.py#L308).

I'm personally not sure how we should proceed 😄

Reply:
It looks like we could jsut default the `learning_rate` to `None`, and then adjust the logic here: https://github.com/PyTorchLightning/lightning-flash/blob/4c624823d7992aa454b9f064d2714cae9b4a8893/flash/core/model.py#L463

If learning rate is None then the `optimizer_kwargs` should just be an empty dict. That would do it I think. Let me know if you'd like to try it otherwise I can take it :smiley:

Reply:
Feel free to take it. Thanks for your support 🙏